dbt/include/teradata/macros/catalog.sql

```diff
@@ -1,7 +1,49 @@
+{% macro teradata__get_views_from_relations(relations) -%}
+    {% set view_relations = [] %}
+    {%- for relation in relations -%}
+        {% if relation.schema and relation.identifier %}
+            {% call statement('check_table_or_view', fetch_result=True) %}
+                SELECT TableKind FROM DBC.TablesV WHERE DatabaseName = '{{ relation.schema }}' AND TableName = '{{ relation.identifier }}'
+            {% endcall %}
+            {% set table_kind = load_result('check_table_or_view').table.columns['TableKind'].values()[0] %}
+            {%- if table_kind == 'V ' -%}
+                {% do view_relations.append(relation) %}
+            {%- endif -%}
+        {% endif %}
+    {%- endfor %}
+    {{ return(view_relations) }}
+{%- endmacro %}
+
+{% macro teradata__create_tmp_tables_of_views(view_relations) -%}
+    {% set view_tmp_tables_mapping = {} %}
+    {%- for relation in view_relations -%}
+        {% set temp_relation_for_view = relation.identifier ~ '_viw_tbl' %}
```

Comment:
as mentioned previously, please add temp label to temporary table name

Reply:
Done, acknowledged above.